packages/utils/src/extendedProfileTypes.ts

```diff
@@ -118,9 +118,9 @@ export type CeramicEPObjects = EPObjects & {
 
 export type HasuraStringProps = HasuraBPStrings & HasuraEPStrings;
 export type HasuraImageSourcedProps = {
-  -readonly [key in keyof typeof BasicProfileImages]?: ImageSources;
+  -readonly [key in keyof typeof BasicProfileImages]?: ImageSources | null;
```

Comment:
That you managed to muddle through the morass that is the Ceramic update code is a miracle to me.

Kudos on getting it done.

Reply:
Thanks for your kind words and the review @dysbulic 🙂